conf/rockstor-scheduling.service

```
@@ -0,0 +1,19 @@
+[Unit]
+Description=Rockstor Scheduling Service
+After=network.target rockstor-pre.service rockstor.service
+Requires=rockstor-pre.service rockstor.service
```

```suggestion
After=rockstor.service
Requires=rockstor.service
```

Network and pre are implied by rockstor.service.